packages/dds/merge-tree/src/test/attributionPolicy.spec.ts

```diff
@@ -185,7 +185,7 @@ describe("Attribution Policy", () => {
 				remoteUserLongId,
 			);
 			client.insertTextRemote(1, "2", { bar: 1 }, ++seq, seq - 1, remoteUserLongId);
-			client.annotateRangeRemote(0, 1, { foo: null }, ++seq, seq - 1, remoteUserLongId);
+			client.annotateRangeRemote(0, 1, { foo: undefined }, ++seq, seq - 1, remoteUserLongId);
```

Comment:
Sanity check: are `undefined` and `null` treated the same by `annotateRangeRemote`?